components/FlexibleHeader/docs/typical-use-scroll-view-observation.md

```diff
@@ -6,16 +6,29 @@ forward UIScrollViewDelegate events to the flexible header by enabling
 scroll view allows the flexible header to over-extend, if enabled, and allows the header's shadow to
 show and hide itself as the content is scrolled.
 
+**Note:** if you support pre-iOS 11 then you will also need to explicitly clear your tracking scroll
+view in your deinit/dealloc method.
+
 <!--<div class="material-code-render" markdown="1">-->
 #### Swift
 ```swift
 flexibleHeaderViewController.headerView.observesTrackingScrollViewScrollEvents = true
+
+deinit {
+  // Required for pre-iOS 11 devices because we've enabled observesTrackingScrollViewScrollEvents.
+  appBarViewController.headerView.trackingScrollView = nil
```

Comment:
Should this documentation be added to AppBar as well or does it automatically take care of this behavior? I don't see any guidance there.

Reply:
Ah thank you. Updated AppBar docs as well.